### PR TITLE
[new locale] mt: Maltese language

### PR DESCRIPTION
**Pull request by alesma**
Months and days names, long and short format, time format for the Maltese Language

### CODE REVIEW

**Comment by alesma** [comment]
About the build error: https://github.com/rollup/rollup/issues/1587

File: src/locale/mt.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        nextDay : '[Għada fil-]LT',
+        nextWeek : 'dddd [fil-]LT',
+        lastDay : '[Il-bieraħ fil-]LT',
+        lastWeek : 'dddd [li għadda] [fil-]LT',
+        sameElse : 'L'
+    },
+    relativeTime : {
+        future : 'f’ %s',
+        past : '%s ilu',
+        s : 'ftit sekondi',
```
Comment (JLHwung) [inline src/locale/mt.js]: Could you add `ss` for `relativeTime`?

Comment (alesma) [reply src/locale/mt.js]: Hi, thanks for the heads up. Just added.

**Comment by ichernev** [comment]
@alesma  thank you!
@JLHwung thank you too :)

**Pull request merged**